components/pkg-dockerize/bin/hab-pkg-dockerize.sh

```diff
@@ -139,8 +139,8 @@ EXPOSE 9631 $(package_exposes $1)
 ENTRYPOINT ["/init.sh"]
 CMD ["start", "$1"]
 EOT
-  docker build --force-rm --no-cache -t $version_tag .
-  docker tag $version_tag $latest_tag
+  docker build --force-rm --no-cache -t ${version_tag,,} .
```

Comment:
Updated the PR to document where downcasing is documented in the Bash manual. Found it on StackOverflow ;)